src/pyorderly/outpack/location_driver.py

```diff
@@ -14,12 +14,24 @@ class LocationDriver(AbstractContextManager):
     """
 
     @abstractmethod
-    def list(self) -> dict[str, PacketLocation]: ...
+    def list_packets(self) -> dict[str, PacketLocation]: ...
```

Comment:
Renaming this allows us to use `list` in the type signatures below instead of `builtins.list`.